pkg/kotsadmconfig/validation/config.go

```diff
@@ -68,12 +68,6 @@ func validateConfigItem(item kotsv1beta1.ConfigItem) (*configtypes.ConfigItemVal
 		return nil, errors.Wrap(err, "failed to get validatable value")
 	}
 
-	// if item is not required and value is empty, no need to validate
-	// if item is required and value is empty, need to validate to return error
-	if validatableValue == "" && !item.Required {
-		return nil, nil
-	}
-
```

Comment:
moved from pkg/kotsadmconfig/validation/config.go so all the logic for isValidatableConfigItem is at one place